pages/quiz-page/quiz-page.css

```diff
@@ -0,0 +1,92 @@
+@import url(/pages/color.css);
+@import url(/style.css);
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Roboto+Condensed&display=swap');
+
+:root {
+    --white-less2: rgb(243, 241, 241);
+}
+
+body {
+    font-family: 'Lato', sans-serif;
+    font-family: 'Roboto Condensed', sans-serif;
+    background: rgb(250, 238, 240);
+}
+
+.header-nav {
+    height: 5rem;
+}
+
+.ques1 {
+    background: var(--white-less2);
+    border-radius: 5px;
+    margin: auto;
+    margin-top: 10rem;
+    width: 35rem;
+    border: solid 2px var(--white-light);
+    padding: 1rem;
+    box-shadow: rgba(175, 78, 92, 0.2) 0px 10px 36px 0px, rgba(190, 103, 116, 0.2) 0px 0px 0px 1px;
+}
+
+.ques1>* {
+    margin: 1rem;
+}
+
+.title {
+    color: var(--pink-primary2);
+    margin-bottom: 2rem;
+}
+
+.ques-score {
+    display: flex;
+    justify-content: space-between;
+    margin-right: 3rem;
+    font-size: 18px;
+}
+
+.ques h2 {
+    font-size: 22px;
+    font-weight: 500;
+}
+
+.options {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.options button {
+    font-size: 18px;
+    width: 90%;
+    height: 2rem;
+    border-radius: 5px;
+    margin-bottom: 14px;
+    border: none;
+    cursor: pointer;
+    border: 1px solid var(--grey);
```

Comment:
you are giving border:none as well as border 1px. Fix it ?

Reply:
I have made the change, thankyou for review